src/assets/stylesheet/index.scss

```diff
@@ -27,42 +31,49 @@ body {
   box-shadow: 1px 3px 7px -3px rgba(0, 0, 0, 0.68);
   border-radius: 6px;
   width: $app-width;
-  margin: 11% auto;
+  margin: 6% auto;
+  display: flex;
+  flex-flow: column;
+  justify-content: flex-start;
+  align-items: center;
 
   @include mobile {
     width: $app-sm-width;
   }
 
   .display {
-    width: $app-width - 2 * $app-padding;
-    padding: 10px $app-padding;
+    width: $app-width - 2 * $display-padding;
+    padding: 10px $display-padding;
     font-size: 3rem;
-    font-weight: 400;
-    min-height: 10rem;
-    background-color: wheat;
-    opacity: 30%;
+    font-weight: 700;
+    min-height: $display-height;
+    background-color: gray;
+    opacity: 40%;
     display: flex;
     justify-content: flex-end;
     align-items: flex-end;
     flex-flow: row wrap;
 
     @include mobile {
-      width: $app-sm-width - 2*$app-sm-padding;
-      padding: 10px $app-sm-padding;
+      width: $app-sm-width - 2*$display-sm-padding;
+      padding: 10px $display-sm-padding;
+      font-size: 2rem;
     }
   }
 
   .button {
-    border: 0px solid lightgrey;
+    border: 0.015px solid lightgrey;
```

Comment:
very narrow hahaha...from the graphics, i was about to say you dont have borders hahaha